### ISSUE TITLE
Missing support GHC 8

### ISSUE THREAD

**Issue by asr** [seereason/process-extras#5]
I couldn't install the library using GHC 8.0.1 RC1.

**Comment by ddssff** [comment]
What happened?

**Comment by asr** [comment]
```
$ cabal install
...
src/System/Process/Common.hs:48:10: error:
    Duplicate instance declarations:
      instance NFData ExitCode
        -- Defined at src/System/Process/Common.hs:48:10
      instance [safe] NFData ExitCode -- Defined in ‘Control.DeepSeq’
Failed to install process-extras-0.3.3.5
```

**Comment by ddssff** [comment]
That's an easy one, but I'd like to make sure that is the only problem.  Could you edit out that instance and try building again?

**Comment by ddssff** [comment]
Its odd because I'm using almost the exact same version of deepseq (1.4.1.1 vs 1.4.1.2) and I don't think I get that error.

**Comment by asr** [comment]
This is the only problem:

```bash
$ cabal unpack process-extras
$ cd process-extras
$ cabal sandbox init
$ cabal install
...
src/System/Process/Common.hs:22:1: warning:
    The import of ‘Control.DeepSeq’ is redundant
      except perhaps to import instances from ‘Control.DeepSeq’
    To import instances alone, use: import Control.DeepSeq()

src/System/Process/Common.hs:40:1: warning:
    The import of ‘GHC.Generics’ is redundant
      except perhaps to import instances from ‘GHC.Generics’
    To import instances alone, use: import GHC.Generics()
...
Installed process-extras-0.3.3.5
```

**Comment by ddssff** [comment]
Is ghc-8.0 really using deepseq-1.4.1.2?

**Comment by asr** [comment]
My version of deepseq is

```bash
$ ghc-pkg list | grep deepseq
deepseq-1.4.2.0
```

**Comment by asr** [comment]
> Is ghc-8.0 really using deepseq-1.4.1.2?

No.

**Comment by ddssff** [comment]
Fixed in version 0.3.3.6.

**Comment by asr** [comment]
Thanks!
